src/SprykerEco/Zed/Braintree/Communication/Plugin/Checkout/BraintreeCheckoutPostSaveHookPlugin.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Zed\Braintree\Communication\Plugin\Checkout;
+
+use Generated\Shared\Transfer\CheckoutResponseTransfer;
+use Generated\Shared\Transfer\QuoteTransfer;
+use Spryker\Zed\Checkout\Dependency\Plugin\CheckoutPostSaveHookInterface;
```

Comment:
Fixed.

Reply:
Adjusted.

Reply:
I reverted this adjustment, because in suite-nonsplit CI we have for command `vendor/bin/console dev:dependency:find` such error `spryker/checkout-extension | Source dependency should not be listed in composer suggests`